lib/components/sections/featured_section.dart

```diff
@@ -219,7 +219,6 @@ class _FeaturedItem extends StatelessWidget {
 
 class _GradientImage extends StatelessWidget {
   final String? image;
-  final height = kIsWeb ? 800.0 : 290.0;
```

Comment:
Although height was set to 800 for web, it was actually 500 because of the height set in `buildSlider()`